src/citra_qt/main.cpp

```diff
@@ -146,7 +147,8 @@ static void InitializeLogging() {
 #endif
 }
 
-GMainWindow::GMainWindow() : config(new Config()), emu_thread(nullptr) {
+GMainWindow::GMainWindow()
+    : config(new Config()), emu_thread(nullptr), ui(std::make_unique<Ui::MainWindow>()) {
```

Comment:
would this be possible?
```suggestion
    : config(std::make_unique<Config>()), emu_thread(nullptr), ui(std::make_unique<Ui::MainWindow>()) {
```